src/fuzz_introspector/datatypes/function_profile.py

```diff
@@ -69,3 +72,23 @@ def load_func_branch_profiles(
             bp_loaded[new_branch.branch_pos] = new_branch
 
         return bp_loaded
+
+    def load_func_callsite(
```

Comment:
Could we make callsite in plural as it loads up to many callsites?

Reply:
changed